### PR TITLE
apache2ctl: add more information link

### PR DESCRIPTION
**Pull request by marchersimon**
For #5510
Addition to #5539

### CODE REVIEW

**Comment by patricedenis** [comment]
Hi @marchersimon 
I just re-checked all the links into the original pages of apache english entries (refered by in ##5539) and re-format them to the "latest" links (in urls) instead of the "buster" one 
I'm not if I haven't done  the work twice with that PR

**Comment by patricedenis** [comment]
Ok no it's OK as you wrote on the pt pages as I see now

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — COMMENTED** [review]
Erm did you mean to do it on the English page too? I can't see a change for the english one here

**Comment by marchersimon** [comment]
@sbrl that's done over in #5539

**Comment by bl-ue** [comment]
🤷🏻 doesn't matter much

File: pages.pt_BR/linux/apache2ctl.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Interface de controle do servidor web HTTP Apache.
 > Este comando está disponível nas distribuições baseadas em Debian, para as baseadas em RHEL veja `httpd`.
+> Mais informações: <https://manpages.debian.org/latest/apache2/apache2ctl.8.en.html>.
```
Comment (navarroaxel) [inline pages.pt_BR/linux/apache2ctl.md:5]:
```suggestion
> Mais informações: <https://manpages.debian.org/latest/apache2/apache2ctl.8.html>.
```

**Comment by sbrl** [comment]
I see, @marchersimon. Could you update the title of this PR then please to reflect that it updates translations, and not the English page please?

**Pull request merged**